Gems/MiniAudio/Code/Source/Clients/MiniAudioListenerComponentController.cpp

```diff
@@ -111,5 +177,10 @@ namespace MiniAudio
         {
             m_entityMovedHandler.Disconnect();
         }
+
+        if (ma_engine* engine = MiniAudioInterface::Get()->GetSoundEngine())
+        {
+            ma_engine_listener_set_cone(engine, m_config.m_listenerIndex, m_config.m_innerAngleInRadians, m_config.m_outerAngleInRadians, (m_config.m_outerVolume/100.f));
```

Comment:
why divide outer volume by 100?

Reply:
From the MiniAudio documentation:
> When the sound is outside of the outer cone, the attenuation will be set to outerGain in the example above. When the sound is in between the inner and outer cones, the attenuation will be interpolated between 1 and the outer gain.

So we accept a "volume" value that is represented in terms of a percentage, but then it need to be divided by 100 to get it back to a decimal since `ma_engine_listener_set_cone`'s `outerGain` argument expects something between 0 and 1.

The reason we expect a percentage rather than a decimal is to make it more intuitive for the user.